docs/rest_api.md

```diff
@@ -0,0 +1,33 @@
+# REST SDK API
+
```

Comment:
We need to add a description here. 
So far, we can deliver it with the following text:
```
SDK Rest API enables an easy way to communicate with it via HTTP.
So far, it is an experimental feature and is available only for local development.
```